homeassistant/components/rflink/sensor.py

```diff
@@ -354,6 +354,7 @@ def _handle_event(self, event):
 
     async def async_added_to_hass(self) -> None:
         """Register update callback."""
+        await super().async_added_to_hass()
```

Comment:
I think not including that call is intentional.
`RflinkSensor` inherits from `RflinkDevice` but the behaviors of both are different in RFLink (and I suspect that is why that call is not made).
Possibly the inheritance should be modified to solve it.

PS: The integration requires a complete refactoring, but I don't have the capacity to do it.

Reply:
Maybe we van just put all the other code in a new method/function and overwrite it here.
Something like:

In `RflinkDevice` change to:
```python

    async def async_added_to_hass(self):
        """Register update callback."""
        await super().async_added_to_hass()
        await register_entity()

    async def register_entity(self):
        """Register entity."""

        # Remove temporary bogus entity_id if added
        tmp_entity = TMP_ENTITY.format(self._device_id)
        .../...
```

Then, in the `RflinkSensor` just overwrite the `register_entity` with its code. 

Makes it sense?

Reply:
Euh, I think I would be able to change this, but I won't be confident to test it. I'll close this PR since we're going to ignore the new pylint issue

Reply:
Thanks anyway for caring about it.

Reply:
Haha no problem :)